### DISCUSSION TITLE
Novità

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#453] [Idee e proposte]
Buongiorno, ci saranno per caso delle novità da introdurre nel registro per il nuovo anno scolastico? E se sì, si potrebbero sapere in linea di massima?

**Comment by trinko** [comment]
Per ora, posso dire che per la prossima versione 1.5.2 cercherò di inserire la gestione delle notifiche e, se tutto va bene, anche le presenze fuori classe.
